src/components/MomentsSection.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from "prop-types";
+import { useState } from "react";
+import { IoIosArrowDropdown, IoIosArrowDropup } from "react-icons/io";
+
+const MomentsSection = ({
+  className,
+  bgColor,
+  hoverBgColor,
+  textColor,
+  moments,
+}) => {
+  const numItems = moments.length;
+  hoverBgColor = `hover:${hoverBgColor}`;
```

Comment:
this doesn't work, tailwind does not allow string interpolation: https://tailwindcss.com/docs/content-configuration#class-detection-in-depth, don't know why it worked for `bg-purple-700` in this case but if you try other colors they don't work